openapi-v2.yaml

```diff
@@ -506,3 +608,70 @@ paths:
           description: error response
           schema:
             $ref: "#/definitions/Error"
+
+  /arrays/{namespace}/{array}/activity:
+    parameters:
+      - name: namespace
+        in: path
+        description: namespace array is in (an organization name or user's username)
+        type: string
+        required: true
+      - name: array
+        in: path
+        description: name/uri of array that is url-encoded
+        type: string
+        required: true
+      - name: start
+        in: query
+        description: "Start time of window of fetch logs, unix epoch in seconds (default: seven days ago)"
```

Comment:
To match a comment on the implementation of this: Could this be a millisecond timestamp, to match the timestamps of the rest of the TileDB API?

Reply:
In the v1 version of this endpoint the timestamp is in seconds as well, that's why we created `ParseSecondTime`.
`FetchAccessLogsPaginated` receives timestamps in seconds we could make the convertion just for the v2 but i think it will complicate things.

Reply:
Keeping the versions consistent seems like a good idea.